src/pybind/mgr/cephadm/tests/fixtures.py

```diff
@@ -37,16 +37,41 @@ def mon_command(*args, **kwargs):
 @pytest.yield_fixture()
 def cephadm_module():
     with mock.patch("cephadm.module.CephadmOrchestrator.get_ceph_option", get_ceph_option),\
-            mock.patch("cephadm.module.CephadmOrchestrator.remote"),\
-            mock.patch("cephadm.module.CephadmOrchestrator.send_command"), \
-            mock.patch("cephadm.module.CephadmOrchestrator.mon_command", mon_command):
+         mock.patch("cephadm.module.CephadmOrchestrator.remote"), \
+         mock.patch("cephadm.module.CephadmOrchestrator.send_command"), \
+         mock.patch("cephadm.module.CephadmOrchestrator.get_osdmap"), \
+         mock.patch("cephadm.module.CephadmOrchestrator.mon_command", mon_command):
 
         m = CephadmOrchestrator.__new__ (CephadmOrchestrator)
         m.__init__('cephadm', 0, 0)
         m._cluster_fsid = "fsid"
         yield m
 
 
+@pytest.yield_fixture()
+def rm_util():
```

Comment:
```suggestion
def rm_util(cephadm_module):
```

does this work as well?

Reply:
No, unfortunately you can't pass a fixture as a parameter to another fixture. I had the same idea in the beginning :/

Reply:
sigh.